package.json

```diff
@@ -35,32 +35,28 @@
         "@fluid-project/eleventy-plugin-fluid": "github:fluid-project/eleventy-plugin-fluid#ee75b78"
     },
     "devDependencies": {
-        "@babel/cli": "7.8.4",
-        "@babel/core": "7.9.6",
-        "@babel/preset-env": "7.9.6",
         "@commitlint/cli": "11.0.0",
         "@commitlint/config-conventional": "11.0.0",
-        "babel-loader": "8.1.0",
-        "babel-plugin-transform-es2015-modules-commonjs-simple": "6.7.4",
         "cross-env": "7.0.2",
         "debug": "4.2.0",
-        "file-loader": "6.0.0",
-        "fluid-grunt-lint-all": "1.0.8",
+        "fast-glob": "3.2.4",
+        "fluid-grunt-lint-all": "github:jobara/fluid-grunt-lint-all#issue-14",
```

Comment:
Would be preferable to link to an actual release. I spoke with @the-t-in-rtf in the [fluid-work channel today](http://irc-logs.fluidproject.org/%23fluid-work/%23fluid-work.2020-10-15.log) about it. We are just waiting on @amb26 to sign off on https://github.com/fluid-project/infusion/pull/1013. After that we can merge in https://github.com/fluid-project/eslint-config-fluid/pull/8 and https://github.com/fluid-project/fluid-grunt-lint-all/pull/15 (needs a new release of the eslint-config-fluid) and create new releases for both respectively.

Reply:
Agreed.